blog/_posts/2017-08-28-gsoc-connecting-contributors-with-projects.md

```diff
@@ -102,7 +102,7 @@ implicit object ProjectAs extends HitReader[Project] {
                 beginnerIssues = searchHits
                   .getHits()
                   .map { hit =>
-                    nread[GithubIssue](hit.getSourceAsString)
+                    nread[GitHubIssue](hit.getSourceAsString)
```

Comment:
This is probably wrong. I thought I had checked the `GitHub` replacements, but I guess I missed this set...